sample/bilibili.py

```diff
@@ -180,10 +193,35 @@ def get_video_info(self, aid: int = 0, bid: str = None):
             return None
         video_info = json.loads(resp.content.decode('utf-8'))
         if video_info['code'] != 0:
-            logger.info('获取视频详细信息失败，错误码：%d，错误信息1：%s', video_info['code'], video_info['message'])
+            logger.info('获取视频详细信息失败，错误码：%d，错误信息1：%s',
+                        video_info['code'], video_info['message'])
             return None
         else:
             return video_info['data']
 
+    def get_user_video(self, mid:int, pn:int=1, ps:int=25, order:str='', tid:int=None, keyword:str=''):
+        if mid == None:
+            logger.error('必须填写mid')
+            return None
```

Comment:
```suggestion
    def get_user_video(self, mid:int, pn:int=1, ps:int=25, order:str='', tid:int=0, keyword:str='') -> list:
```
不需要判断None，python中缺少实参会直接报错